src/metabase/models/pulse.clj

```diff
@@ -192,7 +193,7 @@
   "Update an existing `Pulse`, including all associated data such as: `PulseCards`, `PulseChannels`, and `PulseChannelRecipients`.
 
    Returns the updated `Pulse` or throws an Exception."
-  [{:keys [id name cards channels] :as pulse}]
+  [{:keys [id name cards channels skip_if_empty?] :as pulse}]
```

Comment:
Yes that should be `skip-if-empty?` as well. There should never be a variable that's `snake_case` with a question mark at the end. We use `snake_case` in a few places as a convenience for the API or because that's how columns are named in the DB. In JS you can do something like

```javascript
request.skip_if_empty = true;
```

But you can't do 

```javascript
request.skip_if_empty? = true;
```

So that doesn't make sense.